src/App.jsx

```diff
@@ -543,14 +542,6 @@ export default class App extends Component {
   };
   
   renderButtons(ref) {
-    const element = (<div><div><TaskPopup smart = {this.smart} />
-    <div className="task-button">
-        <label>Attestation</label>  <input type="checkbox" onChange={()=>{this.setTasks()}} id="attestationCheckbox"></input>
```

Comment:
Are we trying to remove the attestation button as well?

Reply:
Nope, we still need the attestation button. Thanks Keeyan